<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomopyReconWindow</class>
 <widget class="QMainWindow" name="TomopyReconWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TomoPy Reconstruction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="paramsLayout">
        <item>
         <layout class="QGridLayout" name="optionsLayout">
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="maxProjAngle">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="value">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="maxProjAngleLebsl">
            <property name="text">
             <string>Maximum projection angle</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="corLabel">
            <property name="text">
             <string>Rotation centre:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="tiltLabel">
            <property name="text">
             <string>Tilt angle:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="tilt">
            <property name="decimals">
             <number>9</number>
            </property>
            <property name="minimum">
             <double>-6.000000000000000</double>
            </property>
            <property name="maximum">
             <double>6.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="stackSelectorLabel">
            <property name="text">
             <string>Stack:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="StackSelectorWidgetView" name="stackSelector"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="previewSliceLabel">
            <property name="text">
             <string>Preview slice:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="previewSlice"/>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="cor">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="reconButtonGroup">
          <item>
           <widget class="QPushButton" name="reconstructSlice">
            <property name="text">
             <string>Reconstruct Slice</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reconstructVolume">
            <property name="text">
             <string>Reconstruct Volume</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="projectionLayout"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="reconLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StackSelectorWidgetView</class>
   <extends>QComboBox</extends>
   <header>mantidimaging.gui.widgets.stack_selector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
